src/Parser/Monk/Brewmaster/Modules/Spells/IronSkinBrew.js

```diff
@@ -21,17 +21,71 @@ class IronSkinBrew extends Analyzer {
   hitsWithoutIronSkinBrew = 0;
   damageWithoutIronSkinBrew = 0;
 
+  // duration tracking to record clipping of buff
```

Comment:
We would prefer it if properties that are for internal usage are prefixed with an underscore.